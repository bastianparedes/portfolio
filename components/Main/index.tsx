import React from 'react'
import Nav from '../Nav'
import About from '../About'
import Portfolio from '../Portfolio'
import Contact from '../Contact'
import styles from './styles.module.css'

const Main = (): JSX.Element => {
  return (
    <main className={styles.main}>
      <Nav />
      <About />
      <Portfolio />
      <Contact />
    </main>
  )
}

export default Main
