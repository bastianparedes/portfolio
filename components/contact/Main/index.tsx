import React from 'react';

import styles from './styles.module.scss';

interface props {
  children?: React.ReactNode;
}

const Main = ({ children }: props): JSX.Element => {
  return (
    <main className={styles.main}>
      <h1 className={styles.title}>ContÃ¡ctame</h1>
      {children}
    </main>
  );
};

export default Main;
