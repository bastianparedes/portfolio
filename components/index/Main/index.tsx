import React from 'react';

import About from '../About';
import Contact from '../Contact';
import Nav from '../Nav';
import Portfolio from '../Portfolio';
import styles from './styles.module.scss';

const Main = (): JSX.Element => {
  return (
    <main className={styles.main}>
      <Nav />
      <About />
      <Portfolio />
      <Contact />
    </main>
  );
};

export default Main;
