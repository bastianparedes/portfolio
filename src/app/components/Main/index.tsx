import React from 'react';

import styles from './styles.module.scss';
import About from '../About';
import Contact from '../Contact';
import Nav from '../Nav';
import Portfolio from '../Portfolio';

const Main = () => {
  return (
    <main className={styles.main}>
      <Nav />
      <About />
      <Portfolio />
      <Contact />
    </main>
  );
};

export default Main;
