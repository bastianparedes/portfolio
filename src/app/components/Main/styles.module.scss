.main {
  position: relative;
  background-color: rgb(0, 85, 124);
}
