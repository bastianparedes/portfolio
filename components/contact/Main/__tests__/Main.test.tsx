import React from 'react';

import { render } from '@testing-library/react';

import Main from '..';

describe('<Main />', () => {
  it('should render', () => {
    const { container } = render(<Main />);
    expect(container).toMatchSnapshot();
  });
});
