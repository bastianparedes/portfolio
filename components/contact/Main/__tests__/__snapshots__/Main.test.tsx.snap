// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> should render 1`] = `
<div>
  <main
    class="main"
  >
    <h1
      class="title"
    >
      Cont√°ctame
    </h1>
  </main>
</div>
`;
